reference/5-api-publisher/Considerations-docker-configuration-and-execution.md

```diff
@@ -0,0 +1,171 @@
+# Running in Docker Desktop.
+
+Docker Containers have the added benefit of running anywhere (e.g. VMs, on-premises in the cloud), which is a massive advantage for both developers and deployment. Leading cloud providers, including Google Cloud, Amazon Web Services (AWS) and Microsoft Azure have adopted it. For simplicity the steps below describe how to use Docker Compose to deploy the Ed-Fi Api Publisher and related tools on Docker Desktop.
+
+# Setup
```

Comment:
There should only be one top level heading in a file. This heading does not serve a purpose, and the line can be removed.

Reply:
fixed